app/controllers/users/review_requests_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Users::ReviewRequestsController < ApplicationController
+  def index
+    @room = Room.find(params[:room_id])
+    @open_state = params[:open_state].presence || "open"
+    @review_reqs =
+      case @open_state
+      when "open"
+        @room.review_assigns.where(is_open: true)
+      when "closed"
+        @room.review_assigns.where(is_open: false)
+      when "all"
+        @room.review_assigns
+      end
+  end
+
+  def create
```

Comment:
__[RuboCop]__

Metrics/AbcSize: Assignment Branch Condition size for create is too high. [26.94/24]


<!-- Comment by [Sider](https://sider.review) -->